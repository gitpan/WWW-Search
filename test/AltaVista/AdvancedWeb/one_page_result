http://www.altavista.digital.com/cgi-bin/query?what=web&text=yes&fmt=d&pg=aq&q=LSAM+and+AutoSearch
