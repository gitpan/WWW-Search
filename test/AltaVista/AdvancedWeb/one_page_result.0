<html><head>
<title>AltaVista: Advanced Query LSAM and AutoSearch</title>
</head><body>
<a href="/cgi-bin/query?pg=q&text=yes"><b>AltaVista Search</b></a><br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=aq>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1 face="arial, helvetica"><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cz >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><textarea name=q cols=65 rows=3 wrap=virtual>LSAM and AutoSearch</textarea><br><a href="/av/content/help_advanced.htm#ranking"><b>Ranking:</b></a><input type=text name=r size=25 maxlength=800 value="">
<b><a href="/av/content/help_advanced.htm#dates" title="Keep documents published after this date"><font size=-1 face="arial, helvetica"><b> From:</b></font></a> <INPUT TYPE=text NAME=d0 SIZE=9 MAXLENGTH=32 VALUE="21/Mar/86">
<b><a href="/av/content/help_advanced.htm#dates" title="Keep documents published before this date"><font size=-1 face="arial, helvetica"><b> To:</b></font></a> <INPUT TYPE=text NAME=d1 SIZE=9 MAXLENGTH=32 VALUE=""><br><INPUT TYPE=checkbox name=fmt value=n> <b>Give me only a precise count of matches.</b> 
<br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_advanced.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=aq&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=q&text=yes">Simple Search</a><br><BR><font size=-1 face="arial, helvetica"><font color=red>Click to find related books at </font><a href="http://www.amazon.com/exec/obidos/av-search/q=LSAM+and+AutoSearch"><b>Amazon.com</b>.</a><br><b>5</b> documents match your query.</font><BR><dl>
<dt><font size=-1><b>1. </b></font><a href="http://www2.ncsu.edu/eos/contrib/perl/bin/AutoSearch"><strong>first_date.html empty page to establish weekly file format</strong></a><dd>afs/eos.ncsu.edu/contrib/perl5004/bin/perl -w eval 'exec perl -S $0 "$@"' if 0; # # AutoSearch.pl # Copyright (c) 1996-1997 University of Southern...<br><cite><a href="http://www2.ncsu.edu/eos/contrib/perl/bin/AutoSearch">http://www2.ncsu.edu/eos/contrib/perl/bin/AutoSearch</a> - size 52K - 10-Jun-97 - English</cite><br>
<P><dt><font size=-1><b>2. </b></font><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html"><strong>Test Pages for AutoSearch</strong></a><dd>Test Pages for AutoSearch. Singleton. Dozen: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12. hidden anchor page. AutoSearch-page. Page maintainer: John Heidemann...<br><cite><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html">http://www.isi.edu/lsam/autosearch/TEST/index.html</a> - size 1K - 7-Oct-97 - English</cite><br>
<P><dt><font size=-1><b>3. </b></font><a href="http://gost.isi.edu/people/johnh/WORK/"><strong>John Heidemann's Work</strong></a><dd>John Heidemann's Work. I am computer scientist at Information Sciences Institute of the University of Southern California. I will also be teaching at USC..<br><cite><a href="http://gost.isi.edu/people/johnh/WORK/">http://gost.isi.edu/people/johnh/WORK/</a> - size 6K - 25-Jun-97 - English</cite><br>
<P><dt><font size=-1><b>4. </b></font><a href="http://www.isi.edu/lsam/tools/WWW_SEARCH/"><strong>WWW::Search</strong></a><dd>WWW::Search. WWW::Search is a collection of Perl modules which provide an API to WWW search engines like AltaVista, Lycos, Hotbot, WebCrawler, and so on...<br><cite><a href="http://www.isi.edu/lsam/tools/WWW_SEARCH/">http://www.isi.edu/lsam/tools/WWW_SEARCH/</a> - size 3K - 8-Oct-97 - English</cite><br>
<P><dt><font size=-1><b>5. </b></font><a href="http://www.isi.edu/lsam/tools/autosearch/"><strong>index.html directory page</strong></a><dd>AutoSearch WEB Searching. What is AutoSearch? AutoSearch performs a web-based search and puts the results set in a web page. It periodically updates this..<br><cite><a href="http://www.isi.edu/lsam/tools/autosearch/">http://www.isi.edu/lsam/tools/autosearch/</a> - size 2K - 3-Feb-97 - English</cite><br>
<P></dl>
<font size=-1 face="arial, helvetica"><font size=-1></font>
</font><P><br>
<HR><font face="arial, helvetica" size=-1><a href="/av/content/av_network.htm">Our Network</a> | <a href="/av/content/addurl.htm">Add/Remove URL</a> | <a href="/av/content/questions.htm">Feedback</a> | <a href="/av/content/help.htm">Help</a><br><a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | <a href="/av/content/about.htm">About AltaVista</a> | <a href="/av/content/jobs.htm">Jobs</a> | <a href="/cgi-bin/query?pg=aq">Graphics</a></font><HR><a href="/av/content/disclaimer.htm">Disclaimer</a> | <a href="/av/content/privacy.htm">Privacy Statement</a><br><a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1997. All Rights Reserved</FORM>
</BODY></HTML>
