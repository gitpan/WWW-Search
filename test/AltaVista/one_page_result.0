<html><head>
<title>AltaVista: Simple Query +LSAM +AutoSearch</title>
</head><body>
<a href="/cgi-bin/query?pg=q&text=yes"><b>AltaVista Search</b></a><br>
<FORM name=mfrm method=GET action="/cgi-bin/query">
<INPUT TYPE=hidden NAME=pg VALUE=q>
<INPUT TYPE=hidden NAME=text VALUE=yes>
<font size=-1 face="arial, helvetica"><B>Search <SELECT NAME=what><OPTION VALUE=web SELECTED>the Web<OPTION VALUE=news >Usenet</SELECT>
for documents in <SELECT NAME=kl><OPTION VALUE=XX SELECTED>any language 
<OPTION VALUE=zh >Chinese
<OPTION VALUE=cz >Czech
<OPTION VALUE=da >Danish
<OPTION VALUE=nl >Dutch
<OPTION VALUE=en >English
<OPTION VALUE=et >Estonian
<OPTION VALUE=fi >Finnish
<OPTION VALUE=fr >French
<OPTION VALUE=de >German
<OPTION VALUE=el >Greek
<OPTION VALUE=he >Hebrew
<OPTION VALUE=hu >Hungarian
<OPTION VALUE=is >Icelandic
<OPTION VALUE=it >Italian
<OPTION VALUE=ja >Japanese
<OPTION VALUE=ko >Korean
<OPTION VALUE=lv >Latvian
<OPTION VALUE=lt >Lithuanian
<OPTION VALUE=no >Norwegian
<OPTION VALUE=pl >Polish
<OPTION VALUE=pt >Portuguese
<OPTION VALUE=ro >Romanian
<OPTION VALUE=ru >Russian
<OPTION VALUE=es >Spanish
<OPTION VALUE=sv >Swedish
</SELECT></B></font><br><INPUT NAME=q size=65 maxlength=800 wrap=virtual VALUE="+LSAM +AutoSearch"><br><INPUT TYPE=radio NAME=act VALUE=search CHECKED>search <INPUT TYPE=radio NAME=act VALUE=refine>refine <INPUT TYPE=submit VALUE=Submit>
 <BR><a href="/av/content/help_simple.htm">Help</a> . <a href="/cgi-bin/query?pg=tmpl&v=pref.html&text=yes">Preferences</a> . <a href="/cgi-bin/query?pg=q&text=yes">New Search</a> . <a href="/cgi-bin/query?pg=aq&text=yes">Advanced Search</a><br><BR><font size=-1 face="arial, helvetica"><b>3</b> documents match your query.</font><BR><dl>
<dt><font size=-1><b>1. </b></font><a href="http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html"><strong>AutoSearch -- a web-searching application</strong></a><dd>AutoSearch. NAME. SYNOPSIS. DESCRIPTION. OPTIONS. DESCRIPTION. AUTOMATED SEARCHING. CHANGING THE LOOK OF THE PAGES. FILES. SEE ALSO. AUTHORS. COPYRIGHT....<br><cite><a href="http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html">http://www.engelschall.com/e/perldoc/pages/util/AutoSearch.html</a> - size 16K - 29-May-97 - English</cite><br>
<P><dt><font size=-1><b>2. </b></font><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html"><strong>Test Pages for AutoSearch</strong></a><dd>Test Pages for AutoSearch. Singleton. Dozen: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12. hidden anchor page. AutoSearch-page. Page maintainer: John Heidemann...<br><cite><a href="http://www.isi.edu/lsam/autosearch/TEST/index.html">http://www.isi.edu/lsam/autosearch/TEST/index.html</a> - size 1K - 5-Aug-97 - English</cite><br>
<P><dt><font size=-1><b>3. </b></font><a href="http://gost.isi.edu/~johnh/WORK/"><strong>John Heidemann's Work</strong></a><dd>John Heidemann's Work. I am computer scientist at Information Sciences Institute of the University of Southern California. I will also be teaching at USC..<br><cite><a href="http://gost.isi.edu/~johnh/WORK/">http://gost.isi.edu/~johnh/WORK/</a> - size 5K - 1-Apr-97 - English</cite><br>
<P></dl>
<b>Tip: </b>

To exclude a word, add a minus sign directly in front of it:
<b>eclipse -sun</b>
<br><BR><font size=-1 face="arial, helvetica"><font size=-1>Word count: LSAM: 92; AutoSearch: 369<BR></font>
</font><P><br>
<HR><font face="arial, helvetica" size=-1><a href="/av/content/av_network.htm">Our Network</a> | <a href="/av/content/addurl.htm">Add/Remove URL</a> | <a href="/av/content/questions.htm">Feedback</a> | <a href="/av/content/help.htm">Help</a><br><a href="http://www.doubleclick.net/jumps/jump1217.htm">Advertising Info</a> | <a href="/av/content/about.htm">About AltaVista</a> | <a href="/av/content/jobs.htm">Jobs</a> | <a href="/cgi-bin/query?pg=q">Graphics</a></font><HR><a href="/av/content/disclaimer.htm">Disclaimer</a> | <a href="/av/content/privacy.htm">Privacy Statement</a><br><a href="http://www.digital.com"><b>DIGITAL</b></a> Copyright 1997. All Rights Reserved</FORM>
</BODY></HTML>
