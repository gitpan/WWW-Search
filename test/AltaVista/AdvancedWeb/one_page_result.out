http://www2.ncsu.edu/eos/contrib/perl/bin/AutoSearch
http://www.isi.edu/lsam/autosearch/TEST/index.html
http://gost.isi.edu/people/johnh/WORK/
http://www.isi.edu/lsam/tools/WWW_SEARCH/
http://www.isi.edu/lsam/tools/autosearch/
